inference-engine/src/transformations/src/transformations/convert_opset1_to_legacy/convert_opset1_to_legacy.cpp

```diff
@@ -109,6 +110,10 @@ bool ngraph::pass::ConvertOpSet1ToLegacy::run_on_function(std::shared_ptr<ngraph
     manager.register_pass<ngraph::pass::GroupConvolutionBackpropDataMultiplyFusion>();
     manager.register_pass<ngraph::pass::ConstantFolding>();
 
+    // Multiply the thrird and fourth input instead of the output of FQ with all const inputs
+    manager.register_pass<ngraph::pass::FakeQuantizeMulFusion>();
+    manager.register_pass<ngraph::pass::ConstantFolding>();
```

Comment:
@GlebKazantaev please confirm that this is the right place

Reply:
As it is pure opset4 transformation and it's intention is to optimize graph I suggest to move it to CommonOptimizations.

Reply:
Decided to keep it in this place as it depends on Conv->Mul fusion that still locates in legacy conversion.